app/assets/javascripts/discourse/tests/acceptance/sidebar-narrow-desktop-test.js

```diff
@@ -0,0 +1,52 @@
+import { test } from "qunit";
+
+import { click, settled, visit, waitUntil } from "@ember/test-helpers";
+
+import { acceptance, exists } from "discourse/tests/helpers/qunit-helpers";
+
+acceptance("Sidebar - Narrow Desktop", function (needs) {
+  needs.user();
+
+  needs.settings({
+    enable_experimental_sidebar_hamburger: true,
+    enable_sidebar: true,
+  });
+
+  needs.hooks.beforeEach(function () {
+    window.innerWidth = 1280;
+  });
+  needs.hooks.afterEach(function () {
+    $("body").width(1280);
```

Comment:
We should avoid using jQuery :wink:. Also, is there a way for us to detect the original width and reset to the original width instead?

Reply:
yay, style width is also triggering ResizeObserver so I can avoid jQuery :) I will wait for CI to be sure it is happy :)